src/Symfony/Component/Yaml/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+5.0.0
+-----
+
+ * The parser is now stricter and will trigger a deprecation notice when a 
```

Comment:
I would reword this paragraph like this:

> Using a mapping inside a multi-line string is deprecated and will throw a `ParseException` in 5.0.

Reply:
Done